### PR TITLE
chore: adding dart docs

### PR DESCRIPTION
**Pull request by pgautier404**

### CODE REVIEW

File: docs/cache/develop/sdks/dart/cheat-sheet.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 1
+sidebar_label: Cheat Sheet
+sidebar_class_name: sidebar-item-swift
```
Comment (cprice404) [inline docs/cache/develop/sdks/dart/cheat-sheet.mdx]: should this say dart

Comment (pgautier404) [reply docs/cache/develop/sdks/dart/cheat-sheet.mdx]: yes, i had some stashed commits i need to add in. sorry for the false alarm.

File: i18n/ja/docusaurus-plugin-content-docs/current/cache/develop/sdks/dart/cheat-sheet.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
```
Comment (cprice404) [inline i18n/ja/docusaurus-plugin-content-docs/current/cache/develop/sdks/dart/cheat-sheet.mdx]: i think we are not supposed to commit stuff into the i18n directory, I think the translator bot is supposed to do that. i'd probably leave this stuff out for now and we can ask erika to take a peek? there is already a bit of backlog for getting the translations up to date.

but if you know things about this directory that i don't, lmk!

**Review by cprice404 — COMMENTED** [review]
❤️

File: i18n/ja/docusaurus-plugin-content-docs/current/cache/develop/sdks/dart/cheat-sheet.mdx (hunk 2 above)
Comment (pgautier404) [reply i18n/ja/docusaurus-plugin-content-docs/current/cache/develop/sdks/dart/cheat-sheet.mdx]: Ok, I'm happy to remove these. I wasn't sure how this process was supposed to work.

**Review by cprice404 — APPROVED** [review]

**Pull request merged**